content/ru/distributed-apps.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Распределенные приложения
+status: Completed
+category: concept
+tags: ["architecture", "", ""]
+---
+
+Распределенное приложение — это приложение, функциональность которого разделена на множество более мелких независимых частей. 
+Распределенные приложения обычно состоят из отдельных [микросервисов](/microservices-architecture/), которые решают различные задачи. 
+В облачной среде отдельные компоненты, как правило, выполняются в виде [контейнеров](/ru/container/) в [кластере](/ru/cluster/).
+
+## Какую проблему решает 
+
+Приложение, работающее на одном компьютере — это единая точка отказа: если этот компьютер выходит из строя, приложение становится недоступным.
+Распределенные приложения зачастую противопоставляются [монолитным](/monolithic-apps/).
+Монолитное приложение обычно сложнее масштабировать, так как его компоненты невозможно масштабировать независимо друг от друга.
+Кроме того, по мере своего роста монолитные приложения все сильнее тормозят разработку, поскольку растет число разработчиков, которым приходится работать над общей кодовой базой, а у нее не всегда есть четко определенные границы.
+
+## Как именно решает проблему
+
+Разбивая приложение на части и запуская их на разных машинах, мы повышаем устойчивость системы к сбоям. 
+Этот подход также позволяет воспользоваться способами масштабирования, которые недоступны монолитам, а именно [горизонтальным масштабированием](horizontal-scaling/). 
```

Comment:
#3219 is not merged yet, but it's coming soon, so I think we can link it already.
```suggestion
Этот подход также позволяет воспользоваться способами масштабирования, которые недоступны приложениям с единственным экземпляром, а именно — [горизонтальным масштабированием](/ru/horizontal-scaling/). 
```